<launch>

    <set_env name="RCUTILS_CONSOLE_OUTPUT_FORMAT" value="{function_name}(): {message} "/>
    <set_env name="RCUTILS_COLORIZED_OUTPUT" value="1"/>

    <!-- 変数 -->
    <arg name="name_space" default=""/> <!-- config/{}.yaml内のnamespaceとそろえる必要あり -->
    <arg name="name_pkg" default="dynamixel_handler"/>

    <node pkg="$(var name_pkg)" exec="dynamixel_unify_baudrate_node" name="dyn_unify_baudrate" namespace="$(var name_space)" output="screen">
        <param from="$(find-pkg-share $(var name_pkg))/config/config_dynamixel_unify_baudrate.yaml"/>
    </node>
</launch>